File: exampleapp/src/main/java/net/planner/exampleapp/CalendarInfo.java
package net.planner.exampleapp;

import androidx.annotation.NonNull;

import java.util.Locale;

public class CalendarInfo {
    private int id;
    private String account;
    private String name;
    private boolean selected = true;

    private static int ALL_CALENDARS_ID = -100;

    CalendarInfo(int id, String account, String name) {
        this.id = id;
        this.account = account;
        this.name = name;
    }

    CalendarInfo() {
        this(ALL_CALENDARS_ID, null, null);
    }

    @NonNull @Override
    public String toString() {
        return String.format(Locale.getDefault(), "%s - %s", account, name);
    }

    String toDebugString() {
        return String.format(Locale.getDefault(), "ID: %d; account: %s; name: %s", id, account, name);
    }

    boolean isAllItem() {
        return (id == ALL_CALENDARS_ID);
    }

    public int getId() {
        return id;
    }

    public String getAccount() {
        return account;
    }

    public String getName() {
        return name;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

}

File: exampleapp/src/main/java/net/planner/exampleapp/CalendarInfo.kt
package net.planner.exampleapp

import lombok.Getter
import lombok.Setter
import java.util.*

class CalendarInfo @JvmOverloads internal constructor(
    @field:Getter private val id: Int = ALL_CALENDARS_ID,
    @field:Getter private val account: String? = null,
    @field:Getter private val name: String? = null
) {
    @Setter
    @Getter
    private val selected = true
    override fun toString(): String {
        return String.format(Locale.getDefault(), "%s - %s", account, name)
    }

    fun toDebugString(): String {
        return String.format(
            Locale.getDefault(),
            "ID: %d; account: %s; name: %s",
            id,
            account,
            name
        )
    }

    val isAllItem: Boolean
        get() = id == ALL_CALENDARS_ID

    companion object {
        private const val ALL_CALENDARS_ID = -100
    }
}